package.json

```diff
@@ -54,7 +54,8 @@
   },
   "scripts": {
     "lint": "./node_modules/.bin/eslint --ext js --ext jsx app/javascript/",
-    "test": "jest"
+    "test": "jest",
+    "relative-to-alias": "jscodeshift --extensions jsx -t ./config/transforms/relative_to_alias_path.js ./ "
```

Comment:
I think this can be consolidated with the addition of the transformer script. I'm surprised you didn't have to specify js and jsx in the extensions list. Is js a default?

Reply:
Yeah, js is the default